src/test/parameters.test.ts

```diff
@@ -31,28 +32,28 @@ describe("Parameters", () => {
 
     const options = {
       parameters: {
+        ids: "ids",
         c_validCustomParam: "custom_param",
       },
-      body: { type: "guest" },
     };
 
     nock(`https://${SHORT_CODE}.api.commercecloud.salesforce.com`)
-      .post(
-        `/customer/shopper-customers/v1/organizations/${ORGANIZATION_ID}/customers/actions/login`
+      .get(
+        `/product/shopper-products/v1/organizations/${ORGANIZATION_ID}/products`
       )
       .query({
         siteId: SITE_ID,
-        clientId: CLIENT_ID,
+        ids: "ids",
         c_validCustomParam: "custom_param",
       })
       .reply(200, MOCK_RESPONSE);
 
-    const response = await customersClient.authorizeCustomer(options);
```

Comment:
had to update this unit test to use another endpoint as this deprecated endpoint is now removed